pkgs/top-level/aliases.nix

```diff
@@ -483,16 +483,16 @@ mapAliases ({
   qt-recordmydesktop = throw "qt-recordmydesktop has been removed from nixpkgs, as it's abandoned and uses deprecated libraries"; # added 2019-12-10
   rfkill = throw "rfkill has been removed, as it's included in util-linux"; # added 2020-08-23
   rkt = throw "rkt was archived by upstream"; # added 2020-05-16
-  ruby_2_0_0 = throw "deprecated 2018-0213: use a newer version of ruby";
-  ruby_2_1_0 = throw "deprecated 2018-0213: use a newer version of ruby";
-  ruby_2_2_9 = throw "deprecated 2018-0213: use a newer version of ruby";
-  ruby_2_3_6 = throw "deprecated 2018-0213: use a newer version of ruby";
-  ruby_2_3 = throw "deprecated 2019-09-06: use a newer version of ruby";
-  ruby_2_4_3 = throw "deprecated 2018-0213: use a newer version of ruby";
-  ruby_2_4 = throw "deprecated 2019-12: use a newer version of ruby";
-  ruby_2_5_0 = throw "deprecated 2018-0213: use a newer version of ruby";
-  rubyPackages_2_4 = throw "deprecated 2019-12: use a newer version of rubyPackages instead";
-  rubygems = throw "deprecated 2016-03-02: rubygems is now bundled with ruby";
+  ruby_2_0_0 = throw "ruby_2_0_0 was deprecated in 2018-0213: use a newer version of ruby";
+  ruby_2_1_0 = throw "ruby_2_1_0 was deprecated in 2018-0213: use a newer version of ruby";
+  ruby_2_2_9 = throw "ruby_2_2_9 was deprecated in 2018-0213: use a newer version of ruby";
+  ruby_2_3_6 = throw "ruby_2_3_6 was deprecated in 2018-0213: use a newer version of ruby";
+  ruby_2_3 = throw "ruby_2_3 was deprecated in 2019-09-06: use a newer version of ruby";
+  ruby_2_4_3 = throw "ruby_2_4_3 was deprecated in 2018-0213: use a newer version of ruby";
+  ruby_2_4 = throw "ruby_2_4 was deprecated in 2019-12: use a newer version of ruby";
+  ruby_2_5_0 = throw "ruby_2_5_0 was deprecated in 2018-0213: use a newer version of ruby";
+  rubyPackages_2_4 = throw "rubyPackages_2_4 was deprecated in 2019-12: use a newer version of rubyPackages instead";
+  rubygems = throw "rubygems was deprecated in 2016-03-02: rubygems is now bundled with ruby";
```

Comment:
nit: Except for `ruby_2_4` and `rubyPackages_2_4`, these are days so they should use `on` preposition. You might also want to add a dash between `0213`.

Reply:
fixed, thanks!